repository/w.json

```diff
@@ -23,6 +23,17 @@
 				}
 			]
 		},
+		{
+			"name": "Wakame",
```

Comment:
Many color schemes have "Color Scheme" in their name, makes them easy to identify. You might want to consider doing the same.

Reply:
@braver 
I added "Color Scheme" as a suffix.